.gitlab-ci.yml

```diff
@@ -0,0 +1,59 @@
+stages:
```

Comment:
Please hold off on this one as we are working on this pipelines right now.

IMO if you are adding these jobs to all projects, you should add a different image/dockerfile in infra-dockerfiles and run veracode from there, rather than starting from `advancedtelematic/gitlab-jobs:0.1.0` and always repeating these steps

Reply:
It's not an optimization, it would be better to not pollute every .gitlab-ci.yml file with this

Reply:
That's exactly what it is - we get the same functionality but remove line from .gitlab-ci.yml and move it to dockerfile.

Reply:
not a line, this entire thing can be replaced with one line if it's abstracted into a container:

```   before_script:
    # The latest wrapper version can be found in https://repo1.maven.org/maven2/com/veracode/vosp/api/wrappers/vosp-api-wrappers-java/
    - wget -q -O veracode-wrapper.jar https://repo1.maven.org/maven2/com/veracode/vosp/api/wrappers/vosp-api-wrappers-java/${VERACODE_WRAPPER_VERSION}/vosp-api-wrappers-java-${VERACODE_WRAPPER_VERSION}.jar
    - ./sbt package
    - mkdir -p package
    - mv ota-tuf/target/scala-*/ota_tuf_*.jar ./package/ota_tuf.jar
    - mv libtuf/target/scala-*/libtuf_*.jar ./package/libtuf.jar
    - mv libtuf-server/target/scala-*/libtuf-server_*.jar ./package/libtuf-server.jar
    - mv cli/target/scala-*/cli_*.jar ./package/cli.jar
    - mv keyserver/target/scala-*/keyserver_*.jar ./package/keyserver.jar
    - mv reposerver/target/scala-*/reposerver_*.jar ./package/reposerver.jar
    - zip -r package package
  script:
    - java -jar veracode-wrapper.jar -vid ${VERACODE_API_ID} -vkey ${VERACODE_API_KEY}
      -action UploadAndScan -appname "OTA Backend - ota-tuf" -createprofile true -autoscan true
      -filepath package.zip -version "job ${CI_JOB_ID} in pipeline ${CI_PIPELINE_ID} for ${CI_PROJECT_NAME} repo"
```

Reply:
I don't see how but I'd be delighted to review your PR implementing this. Feel free to tag me for review.

Reply:
I'd be delighted to approve a pr with that.

Reply:
I plan to work on optimising gitlab jobs but no timeline as it's not highest priority ATM.

Reply:
https://github.com/advancedtelematic/ota-tuf/pull/243